test-track/frontend/src/business/plan/view/comonents/api/RelevanceScenarioList.vue

```diff
@@ -257,7 +260,15 @@ export default {
       this.currentPage = 1;
       this.search();
     },
-    search() {
+    pageChange() {
+      this.search("page");
+    },
+    search(data) {
+      if (data !== "page") {
+        this.pageRefresh = false;
```

Comment:
this.pageRefresh = data === "page"；

Reply:
Thank you for your suggestion, it is a good idea